.github/workflows/cross-repo.yaml

```diff
@@ -24,7 +24,6 @@ jobs:
           branch-name: '10.10'
           inputs: |
             branch_name: ${{ github.head_ref || '2.4_10.10' }}
-            hf_version: 56
```

Comment:
No need to have this hardcoded here, because we have a default value in the workflow.